usaspending_api/etl/management/commands/update-file-c-file-d-awards.py

```diff
@@ -0,0 +1,85 @@
+from datetime import datetime
+import logging
+import pandas as pd
+from django.db import connections, transaction, connection
+from usaspending_api.awards.models import Award, FinancialAccountsByAwards
+from usaspending_api.references.models import Agency
+from django.core.management.base import BaseCommand
+# This dictionary will hold a map of tas_id -> treasury_account to ensure we don't
+logger = logging.getLogger('console')
+class Command(BaseCommand):
+    """
+    This command will load a single submission from the DATA Act broker. If
+    we've already loaded the specified broker submisison, this command
+    will remove the existing records before loading them again.
+    """
+    help = "remap file c awards to their corresponding file d award "
+    # def add_arguments(self, parser):
+    #     pass
+    @transaction.atomic
+    def handle(self, *args, **options):
+        remapped_awards = []
+        bad_faba = []
+        logger.info('starting File C awards remapping')
+
+        # 1) Get all file C awards
+        faba_queryset = FinancialAccountsByAwards.objects.filter(award__isnull=False,
+                                                                 # award__latest_transaction_id__isnull=True,
+                                                                 award__recipient_id__isnull=True,
+                                                                 treasury_account__federal_account_id=1411
+                                                                 ).values("piid", "fain", "uri", "award_id",
+                                                                          "financial_accounts_by_awards_id",
+                                                                          "award__awarding_agency__toptier_agency_id")[:1000]
+
+        # # awards that the file D should map to (pandas)
+        # file_d_award_queryset = Award.objects.filter(recipient_id__isnull=False, date_signed__gt='2016-01-01')\
+        #     .values("id", "piid", "fain", "uri","awarding_agency__toptier_agency_id")
+
+        start_time = datetime.now()
+        size_of = str(len(faba_queryset))
+        print(size_of)
+        awards = Award.objects  # this stops the property lookup each iteration, saving 3+ seconds every 100 rows!
+        possible_agencies_cache = {}
+        for index, faba in enumerate(faba_queryset, 1):
+            if not (index % 100):
```

Comment:
How many rows does this typically iterate over? would it be too much to move this to 1000 in order to potentially reduce the possibility of too much logging?